.azure/pipeline/azure-cicd.yml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+#################################################################################
+#                      OneBranch Pipelines - PR Build                           #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
+# Yaml Schema:    https://aka.ms/obpipelines/yaml/schema                        #
+# Retail Tasks:   https://aka.ms/obpipelines/tasks                              #
+# Support:        https://aka.ms/onebranchsup                                   #
+#################################################################################
+
+# https://aka.ms/obpipelines/triggers
+trigger:
+  ## disable batching of changes when a push happens.
+  batch: false
+  branches:
+    include:
+    - main
+    - release/*
+
+pr:
+- main
+- release/*
+
+schedules:
+- cron: '0 21 * * *'
+  displayName: Schedule CI/CD
+  branches:
+    include:
+    - main
+    - release/*
+  always: true
+
+parameters: # parameters are shown up in ADO UI in a build queue time
+- name: 'debug'
+  displayName: 'Enable debug output'
+  type: boolean
+  default: false
+
+variables:
+  CDP_DEFINITION_BUILD_COUNT: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
+  system.debug: ${{ parameters.debug }}
+  ENABLE_PRS_DELAYSIGN: 0
+  ROOT: $(Build.SourcesDirectory)
+  REPOROOT: $(Build.SourcesDirectory)
+  OUTPUTROOT: $(REPOROOT)\out
+  NUGET_XMLDOC_MODE: none
+
+  WindowsContainerImage: 'cdpxwin1809.azurecr.io/global/vse2022:latest' # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+
+jobs:
+
+# regular
+# Always run this job.
+- ${{ if in(variables['Build.Reason'], 'PullRequest', 'BatchedCI', 'IndividualCI', 'Schedule') }}:
+  - template: ./azure-reusable-build.yml
+    parameters:
+      name: regular
+      build_artifact: Build-x64
+      generate_release_package: true
+      build_nuget: true
+      build_options: /p:ReleaseJIT=True
+
+# api_test
```

Comment:
Don't forget to add the other single-machine tests from Invoke-CICDTests:
 "bpftool_tests.exe",
 "sample_ext_app.exe",
 "socket_tests.exe"

Reply:
This PR is as noted in the title, is onboarding api_test, there hasn't a decision made to onboard the "bpftool_tests.exe",
"sample_ext_app.exe", "socket_tests.exe".

Reply:
@delaramamiri if it is not much work to invoke the 3 other single machine tests, please onboard them as part of this PR.